WordPress/Classes/ViewRelated/Blog/Subscribers/SubscriberRowView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+import WordPressUI
+import WordPressKit
+
+struct SubscriberRowView: View {
+    let viewModel: SubscriberRowViewModel
+
+    var body: some View {
+        HStack(alignment: .center) {
+            avatar.frame(width: 24, height: 24)
+            (Text(viewModel.title) + Text(viewModel.details).foregroundColor(.secondary))
+                .lineLimit(1)
+        }
+    }
+
+    @ViewBuilder
+    private var avatar: some View {
+        switch viewModel.avatar {
+        case .remote(let url):
+            AvatarView(style: .single(url), diameter: 24, placeholderImage: Image("gravatar").resizable())
+        case .email:
+            Image(systemName: "envelope")
+                .foregroundStyle(.tertiary)
+        }
+    }
+}
+
+@MainActor
+final class SubscriberRowViewModel: Identifiable {
```

Comment:
Use struct instead of class?

Reply:
I'm planning to add some state to it in the future PRs.

I also often use classes to avoid per-member ARC overhead – been bitten by it before. I'm about to start learning the new [ownership model](https://github.com/swiftlang/swift/blob/main/docs/OwnershipManifesto.md). It should be a better way to avoid ARC while still using value types (with value semantics).